packages/frontend/core/src/components/affine/setting-modal/account-setting/index.tsx

```diff
@@ -145,8 +152,18 @@ const StoragePanel = () => {
   const { data } = useQuery({
     query: allBlobSizesQuery,
   });
+  const [subscription] = useUserSubscription();
+  const plan = subscription?.plan ?? SubscriptionPlan.Free;
+  const maxLimit = plan === SubscriptionPlan.Free ? 10737418240 : 107374182400;
```

Comment:
We can use tool to parse and format bytes https://github.com/visionmedia/bytes.js